src/Template/Bake/Model/table.ctp

```diff
@@ -13,13 +13,30 @@
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
 use Cake\Utility\Inflector;
+
+$annotations = [];
+foreach ($associations as $type => $assocs) {
+    foreach ($assocs as $assoc) {
+        $typeStr = Inflector::camelize($type);
+        $annotations[] = "@property \Cake\ORM\Association\\{$typeStr} \${$assoc['alias']}";
+    }
+}
+$annotations[] = "@method {$namespace}\\Model\\Entity\\{$entity} get(\$primaryKey, \$options = [])";
+$annotations[] = "@method {$namespace}\\Model\\Entity\\{$entity} newEntity(\$data = null, array \$options = [])";
+$annotations[] = "@method {$namespace}\\Model\\Entity\\{$entity}[] newEntities(array \$data, array \$options = [])";
+$annotations[] = "@method {$namespace}\\Model\\Entity\\{$entity}|bool save(Cake\\Datasource\\EntityInterface \$entity, \$options = [])";
```

Comment:
Thanks, I'm going to run the plugin on one of my projects and load up the files in the IDE and make sure it all works as expected.